### ISSUE TITLE
Changing config.defaultIconPack has no effect when importing individual components

### ISSUE THREAD

**Issue by ben-walker** [buefy/buefy#942]
### Overview of the problem

**Buefy** version: 0.6.7-beta.0
**Vuejs** version: 2.5.16
**OS/Browser**: Firefox, Chrome

### Description

I'm trying to import individual components to cut down on my webpack bundle size, which means I can't set constructor options on Buefy initialization.

So I've been trying to do the following to use `fas` as my default icon pack:

`import config from 'buefy/src/utils/config'`
`config.defaultIconPack = 'fas'`

Logging the config elsewhere shows that the defaultIconPack is indeed `fas`, but the element still shows up as `i.mdi` instead of `i.fas`.

The only information I could find regarding this is #73, I'm not sure if the info in that issue still holds true or if the recommended solution has changed; any guidance would be greatly appreciated, thanks!

**Comment by jtommy** [comment]
@ben-walker You're wrong because when you use individual components there isn't constructor (config).
Anyway we'll check and fix each component in order to customize icon pack.

**Comment by ben-walker** [comment]
I understand that there's no constructor when using individual components, I'm wondering if it's possible to update the global defaults without the use of a constructor. If not no worries, I've just been importing and using Buefy the normal way as a fallback.
